autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectColumn.tsx

```diff
@@ -22,12 +25,34 @@ const CourseSelectColumn: React.FC = () => {
   );
   const term = useSelector<RootState, string>((state) => state.termData.term);
   const dispatch = useDispatch();
+  const [loading, setLoading] = React.useState(true);
+  const [wasCourseRemoved, setCourseRemoved] = React.useState(false);
+
+  const removeCallback = React.useCallback((id: number) => {
```

Comment:
```suggestion
  const removeCourseCallback = React.useCallback((id: number) => {
```
Think adding `course` here helps identify what this actually does at a glance

Also, what is the need for this exactly? How come the course cards can't just remove themselves?

Reply:
When a card is removed, it used to dispatch a `removeCourseCard` action themselves. However, now, when a card is removed, it needs to notify the other cards not to animate since we said that we didn't like that animation. This is done by passing the information up to `CourseSelectColumn`, where `wasCourseRemoved` can be set to `true`